source/include/Events.h

```diff
@@ -104,11 +106,11 @@ struct EventData {
     void *pEventData;
 
     EventData() : common(0, 0, RefNumVal()), controlUID(kNotAnEventControlUID), eventDataType(nullptr), pEventData(nullptr) { }
-    EventData(EventSource source, EventType type, const RefNumVal &ref, TypeRef edtype = nullptr, void *pData = nullptr) :
+    EventData(const EventSource source, const EventType type, const RefNumVal &ref, TypeRef edtype = nullptr, void *pData = nullptr) :
```

Comment:
In these three constructors, it seems we are applying const to some parameters and not to others, that is more confusing. We need to be consistent.

Reply:
I think he should not be adding const to any scalar non-ref parameters or locals unless there is a good, conceptual reason, such as a constant used as a static limit or computed size based on sizeof() calculations.
Just because the current usage happens to not modify the value after assignment does not mean it is beneficial to mark it const.

Reply:
Yeah, I will revert these changes.